src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/dto/TestResultsDTO.java

```diff
@@ -36,6 +37,8 @@ public class TestResultsDTO extends AbstractBuildResultNotificationDTO {
 
     private final List<TestSuiteDTO> results;
 
+    // For an unknown reason, the deserialization only works with this annotation
+    @JsonProperty("staticCodeAnalysisReports")
```

Comment:
The `testwiseCoverageReport` has the same comment but no annotation. Should the comment be removed, or the annotation added there?

Reply:
Well, I don't quite know. First, I had no annotation for `staticCodeAnalysisReports`, but the tests failed. After debugging it, I found out, that it was a problem in the convertation to the DTO.
I have also seen the comment and looked into the history of git what annotation was added to `testwiseCoverageReport` (https://github.com/ls1intum/Artemis/pull/5238). This annotation fixed my problem.

So actually, I don't know, since the annotation was removed in a later pull request again (https://github.com/ls1intum/Artemis/pull/5637). (Probably the comment can be removed if it works without the annotation...)